src/SizeMeApp.jsx

```diff
@@ -66,6 +66,7 @@ class SizeMeApp extends React.Component {
                         </ReactTooltip>
                     </div>)}
                 </div>
+                {resolved && document.body.classList.add("sizeme-active")}
```

Comment:
Propably not the best place for this, since the render-function will be called multiple times. Surely it doesn't matter if we add the class to the list multiple times, but it's still not very nice. 
Correct place would be `componentDidMount`-function. Function can be added to the SizeMeApp-class and React will call it once when the component is initialized. 
Also, I think we should remove the class from body when Sizeme is disabled, for example if the user toggles it hidden, right? Then you could add a `componentWillUnmount`-function to SizeMeApp-class and and code to clean the class-attribute there. 
See React component lifecycle here: https://reactjs.org/docs/react-component.html#the-component-lifecycle

Reply:
Furthermore, that `resolved` in SizeMeApp is redundant since SizeMeAppWrapper checks it and doesn't render SizeMeApp at all if `resolved` is false. That's just some leftovers from earlier versions. So, you don't need to check that prop at all, if SizeMeApp is rendered, everything is already resolved.